### PR TITLE
Splits installer and server start, and adds installer test

### PR DESCRIPTION
**Pull request by justafish**
Moved from #73

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
+
 ```bash
 php -r "readfile('https://raw.githubusercontent.com/contentacms/contenta_jsonapi/8.x-1.x/installer.sh');" > contentacms.sh
 chmod a+x contentacms.sh
 ./contentacms.sh
 ```
 
+- To restart the webserver, run `composer run-script start-contenta <install location>`
 - Visit [http://127.0.0.1:8888/](http://127.0.0.1:8888/) and log into your site with `admin`/`test`
+- The host and port can be overridden by copying `.env` to `.env.local`
```
Comment (e0ipso) [inline README.md]: I like this!

Comment (justafish) [reply README.md]: I had to take this out :(
It only works for the quickstart, so there isn't an opportunity to edit it

File: README.md [hunk 2]
```diff
@@ -5,30 +5,39 @@
...
+- Install [composer](https://getcomposer.org/)
+- Install [drush](http://docs.drush.org/en/8.x/install/)
+
+```bash
+$ composer create-project contentacms/contenta-jsonapi-project <DESTINATION> --stability dev --no-interaction
+$ cd <DESTINATION>
 ```
-* After that install Drupal normally.
+
+- Decide whether you want to install with either SQLite `drush si contenta_jsonapi --db-url=sqlite://sites/default/files/.ht.sqlite -y`
```
Comment (dawehner) [inline README.md]: I really like this documentation as it enables people without Drupal knowledge to install it.

**Comment by dawehner** [comment]
Progress!

**Pull request merged**